docs/using-gitbase/examples.md

```diff
@@ -81,9 +81,9 @@ GROUP BY committer_email,
 ```sql
 SELECT
     LANGUAGE(file_path, blob_content) as lang,
-    SUM(JSON_EXTRACT(LOC(file_path, blob_content), '$.Code')) as code,
```

Comment:
I would say to do the opposite, keep output as it was. Also, other functions have keys with first letter uppercase too.

Reply:
Using uppercase is a bit weird in JSON, hence https://github.com/src-d/gitbase/issues/930

If we do want to use uppercase, though, I can close the issue and change it back here.

Reply:
I'm concerned about backward compatibility. All queries that are using these functions will stop working with no error message...